### PR TITLE
fix: zapi should be able to handle signed integers

### PR DESCRIPTION
**Pull request by vgratian**
fix for #387

### CODE REVIEW

File: cmd/collectors/zapi/collector/parsers.go [hunk 1]
```diff
@@ -123,7 +125,11 @@ func (me *Zapi) HandleCounter(path []string, content string) string {
 			me.Logger.Trace().Msgf("%sadd (%s) as instance key [%s]%s => %v", color.Red, key, display, color.End, fullPath)
 		}
 	} else {
-		metric, err := me.Matrix.NewMetricUint64(key)
+		if t := me.Params.GetChildContentS("metric_type"); t != "" {
```
Comment (Hardikl) [inline cmd/collectors/zapi/collector/parsers.go:128]: add one line comment here also

**Review by Hardikl — DISMISSED** [review]
remove Chris's changes from here.

File: cmd/collectors/zapi/README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Parameters different from ZapiPerf:
 
 ### Object configuration file
 
-The Zapi collector does not have the parameters `instance_key` and `override` parameters.
+The Zapi collector does not have the parameters `instance_key` and `override` parameters. The optional parameter `metric_type` allows to override the defualt metric type (`uint64`). The value of this parameter should be only of the metric types supported by [the Matrix data-structure](../../../pkg/matrix/README.md#add-metrics).
```
Comment (cgrinds) [inline cmd/collectors/zapi/README.md:34]: The optional parameter `metric_type` allows you to override the default metric type (`uint64`).

**Review by cgrinds — COMMENTED** [review]
looks good @vgratian

**Comment by vgratian** [comment]
Closing this, re-opened PR: #400. All comments are addressed in the new branch.

**Pull request closed without merging**